react/components/icon/File/index.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { calcIconSize, baseClassname } from '../utils'
+
+const iconBase = {
+  width: 16,
+  height: 16,
+}
+
+const File = ({ color, size, block }) => {
+  const newSize = calcIconSize(iconBase, size)
+
+  return (
+    <svg
```

Comment:
Shouldn't you use the SVG component like the other icon components?

Reply:
good point @igorbrasileiro 

really @vlaux you should use IconBase `import { Svg } from '../ IconBase'`. He is responsible for changing the icons when using admin-style.

Reply:
Thanks for pointing it out, guys. Updated.